public/javascripts/create-password.js

```diff
@@ -0,0 +1,75 @@
+const continueEvent = document.querySelector("#continue-button");
+const passwordFields = document.querySelector("#password-fields");
+const formGroup = document.querySelector(".govuk-form-group");
+const passwordInput = document.querySelector("#password");
+const confirmPasswordInput = document.querySelector("#confirm-password");
+const errorTextSection = document.querySelector("#error");
+const errorSummarySection = document.querySelector("#error-summary");
+
+continueEvent.addEventListener("click", checkPassword);
+
+let passwordMinimumLength = 8;
+let passwordRegex = /^(?=.*[0-9])(?=.*[!@#$%^&*])[a-zA-Z0-9!@#$%^&*]/;
```

Comment:
nice regex!

Reply:
Can't claim any credit. I got it from a blog post 😂

Reply:
No one needs to know 🤫 😂